src/app/tide/active_peptide_queue.cc

```diff
@@ -31,6 +31,7 @@ ActivePeptideQueue::ActivePeptideQueue(RecordReader* reader,
   compiler_prog2_ = new TheoreticalPeakCompiler(&fifo_alloc_prog2_);
   peptide_centric_ = false;
   elution_window_ = 0;
+  exact_pval_search_ = false;
```

Comment:
This variable is used to decide if the score should be scaled. On Ubuntu it is initialized correctly to false by the runtime, but on CentOS it contains garbage and is interpreted as true. Adding explicit initialization.